### PR TITLE
Email address placeholder localization

### PR DESCRIPTION
**Pull request by ljnaresh**
Localize “email address” placeholder -  #20175

### CODE REVIEW

File: i18n/uk.toml [hunk 1]
```diff
@@ -245,3 +245,7 @@ other = "Попередження:"
 [whatsnext_heading]
 # other = "What's next"
 other = "Що далі"
+
+[input_placeholder_email_address]
+# other = "email address"
+other = "електронна адреса"
```
Comment (sftim) [inline i18n/uk.toml:251]: @ljnaresh this file is in (English) alphabetical order. Would you amend this commit to put `[input_placeholder_email_address]` in the same sort order as the English original?

Comment (ljnaresh) [reply i18n/uk.toml]: @sftim, thanks for the suggestion, I have modified my commit with the suggested changes

**Comment by sftim** [comment]
Haven't got Ukrainian localization team approvals sorted yet
/approve

**Pull request merged**